packages/cli/src/lib/setup/setupPacketManager.ts

```diff
@@ -254,6 +261,33 @@ export class PacketManager {
         return failed;
     }
 
+    /**
+     * Upgrade given packets to given version or newest available version
```

Comment:
maybe directly talk about od-packages or such ... if we add nom libs too we also have other packages :-)